Userland/Libraries/LibJS/Parser.cpp

```diff
@@ -4582,4 +4582,7 @@ Parser::ForbiddenTokens Parser::ForbiddenTokens::forbid(std::initializer_list<To
     return result;
 }
 
+template NonnullRefPtr<FunctionExpression> Parser::parse_function_node(u8, Optional<Position> const&);
+template NonnullRefPtr<FunctionDeclaration> Parser::parse_function_node(u8, Optional<Position> const&);
```

Comment:
(as mentioned on discord, this fix looks wrong to me from a distance)

Reply:
Why exactly? Technically, I could have added an `extern template` explicit instantiation declaration to the header, but it works this way too.